### PR TITLE
Friesian log4j fix

### PR DESCRIPTION
**Pull request by cyita**

### CODE REVIEW

File: scala/friesian/pom.xml [hunk 1]
```diff
@@ -217,15 +217,15 @@
             </exclusions>
         </dependency>
         <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-log4j12</artifactId>
-            <version>1.7.16</version>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+            <version>${log4j.version}</version>
```
Comment (glorysdj) [inline scala/friesian/pom.xml:222]: log4j.version is 2.17.1?

File: scala/friesian/pom.xml [hunk 2]
```diff
@@ -217,15 +217,15 @@
...
             <scope>${serving.scope}</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.logging.log4j</groupId>
-            <artifactId>log4j-api</artifactId>
-            <version>${log4j.version}</version>
+            <artifactId>log4j-slf4j-impl</artifactId>
+            <version>2.17.1</version>
```
Comment (glorysdj) [inline scala/friesian/pom.xml:228]: use log4j.version if log4j.version is 2.17.1

File: scala/friesian/pom.xml (hunk 1 above)
Comment (cyita) [reply scala/friesian/pom.xml:222]: Yes

**Comment by cyita** [comment]
related issue https://github.com/intel-analytics/arda-docker/issues/579

**Review by glorysdj — APPROVED** [review]
LGTM

**Comment by cyita** [comment]
jenkins passed http://10.112.231.51:18888/view/ZOO-PR/job/BigDL-PR-Friesian-Validation/238/

**Pull request merged**